packages/wallet/test/services/TransactionsTracker.test.ts

```diff
@@ -183,7 +175,7 @@ describe('TransactionsTracker', () => {
         const tip$ = hot<Cardano.Tip>('----a|', { a: tip });
         const submitting$ = hot('-a---|', { a: tx });
         const pending$ = hot('---a-|', { a: tx });
-        const transactionsSource$ = hot<Cardano.TxAlonzo[]>('-----|');
+        const transactionsSource$ = hot<Cardano.TxAlonzo[]>('-----|') as unknown as TrackerSubject<Cardano.TxAlonzo[]>;
```

Comment:
Just wondering why this was needed, doesn't seem like type of this changed :thinking:

Reply:
My bad, was something I tried but forgot to change it back